lesson_07/conditionals/src/lesson7.ts

```diff
@@ -57,7 +98,11 @@ export function computeFactorial(n: number): number {
  * @return The sum of all the values.
  */
 export function addNumbers(values: number[]): number {
-  return 0;
+  let sum = 0;
+  values.forEach((value) => {
```

Comment:
Would be better to use `values.reduce`. Know how to use it, comes in handy especially for interview prep but also IRL.

Reply:
Thanks ill look into .reduce!!